submissions/laneleka/html-css-popup/index.html

```diff
@@ -0,0 +1,280 @@
+                  </a>
+                </li>
+                <li>
+                  <a class="popup-item" href="#" target="_blank">
+                    <svg>
+                      <use xlink:href="#fest"></use>
+                    </svg>
+                    Cat
+                  </a>
+                </li>
+                <li>
+                  <a class="popup-item" href="#" target="_blank">
+                    <svg>
+                      <use xlink:href="#fest"></use>
+                    </svg>
+                    Cat
+                  </a>
+                </li>
+                <li>
+                  <a class="popup-item" href="#" target="_blank">
+                    <svg>
+                      <use xlink:href="#fest"></use>
+                    </svg>
+                    Cat
+                  </a>
+                </li>
+                <li>
+                  <a class="popup-item" href="#" target="_blank">
+                    <svg>
+                      <use xlink:href="#fest"></use>
+                    </svg>
+                    Cat
+                  </a>
+                </li>
+                <li>
+                  <a class="popup-item" href="#" target="_blank">
+                    <svg>
+                      <use xlink:href="#fest"></use>
+                    </svg>
+                    Cat
+                  </a>
+                </li>
+              </ul>
+            </div>
+          </li>
+          <li tabindex="4">
+            <a tabindex="-1" class="menu-link-user" href="https://github.com/laneleka/kottans-frontend" target="_blank">
+              <span class="hidden">User</span>
+              <div class="user-img"></div>
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+   
+    <video class="background-video" src="./src/cat2.mp4" autoplay muted loop></video>
+    
+  </div>
+  
+  <div class="hidden">
+    <svg xmlns="http://www.w3.org/2000/svg">
+      <symbol id="paw" viewBox="0 0 32 32">
+          <path d="M19.3714 17.3399C18.5552 13.1734 13.2159 13.1868 12.4024 17.3394C11.9832 19.4717 10.2093 19.3433 9.71588 21.5931C9.29547 23.5098 10.7225 24.8096 12.6265 24.8096C12.9786 24.8096 16.0853 24.0994 18.3685 24.7144C19.2271 24.9458 20.5814 24.7701 21.4 23.9886C22.4439 22.992 22.0572 21.5923 22.0509 21.5923C21.5572 19.3433 19.7879 19.4732 19.3714 17.3399V17.3399ZM12.9127 22.9043C12.6138 22.9854 11.3306 22.9829 11.5474 21.9947C11.8497 20.6158 13.6959 20.4803 14.2423 17.7002C14.5308 16.2271 15.5035 16.116 15.7889 16.116C16.8597 16.116 17.3492 16.7701 17.5313 17.6997C18.074 20.4803 19.9166 20.6136 20.2198 21.9947C20.5335 24.0349 16.9656 21.8128 12.9127 22.9043V22.9043Z"/>
+          <path d="M12.989 14.1812C14.9859 14.1812 16.2522 12.0427 15.6138 9.85427C14.7622 6.93532 11.396 6.26223 10.0833 8.64235C8.813 10.9451 10.5083 14.1812 12.989 14.1812ZM11.7251 9.54811C12.5423 8.06691 14.5733 10.2637 13.7107 11.8262C12.8921 13.3091 10.8618 11.1128 11.7251 9.54811Z"/>
```

Comment:
Can you change all your svg to img or not?

Reply:
I read using svg and svg sprite for icons is the best web practice. Could you clarify why do you want to use img?

Reply:
I didn’t write correctly.
Be better if you reduce your html file for increase speed your web-site. In this case you free to do in your own way, but against, be better if you download all your svg to local computer and will be use through relative way.

Reply:
I stay it how it is.